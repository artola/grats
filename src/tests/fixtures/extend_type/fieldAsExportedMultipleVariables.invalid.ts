/** @gqlType */
class SomeType {
  // No fields
}

/** @gqlField */
export const greeting = (_: SomeType): string => {
    return `Hello World`;
  },
  anotherGreeting = (_: SomeType): string => {
    return `Hello World`;
  };
