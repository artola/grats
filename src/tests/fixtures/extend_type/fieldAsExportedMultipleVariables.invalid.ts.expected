-----------------
INPUT
----------------- 
/** @gqlType */
class SomeType {
  // No fields
}

/** @gqlField */
export const greeting = (_: SomeType): string => {
    return `Hello World`;
  },
  anotherGreeting = (_: SomeType): string => {
    return `Hello World`;
  };

-----------------
OUTPUT
-----------------
src/tests/fixtures/extend_type/fieldAsExportedMultipleVariables.invalid.ts:7:1 - error: Expected only one declaration when defining a `@gqlField`, found 2.

  7 export const greeting = (_: SomeType): string => {
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  8     return `Hello World`;
    ~~~~~~~~~~~~~~~~~~~~~~~~~
... 
 11     return `Hello World`;
    ~~~~~~~~~~~~~~~~~~~~~~~~~
 12   };
    ~~~~
